### ISSUE TITLE
Meeting capacity for A1 incorrect

### ISSUE THREAD

**Issue by thewrabbit** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#9896]
The table at the start of this article states that meetings can have up to 20000 participants with 1000 having full interactive capabilities.

However A1 has a limit of 300 interactive meeting participants.

https://learn.microsoft.com/en-us/microsoftteams/limits-specifications-teams#meetings-and-calls

and has no live events capabilities.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 92fcb1f6-5cc2-f6bd-cfa9-11d811f1dbf1
* Version Independent ID: 9a571e6d-8cb3-9ff8-a5c2-62deab146264
* Content: [Meetings, webinars, and live events - Microsoft Teams](https://learn.microsoft.com/en-gb/microsoftteams/quick-start-meetings-live-events)
* Content Source: [Teams/quick-start-meetings-live-events.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/quick-start-meetings-live-events.md)
* Service: **msteams**
* Sub-service: **meetings**
* GitHub Login: @MicrosoftHeidi
* Microsoft Alias: **heidip**

**Comment by get-itips** [comment]
Hello @thewrabbit 
How are you?
The limits and specifications article (that you also link here) **is also linked on this article**, there it is described specific limits for specific scenario like A1.
I don't identify a need to edit this article as the specific limit for A1 is on the linked one. 
Thank you for your feedback,

**Comment by thewrabbit** [comment]
I disagree.  The limits and specifications article is only linked at the bottom as further reading.  There is nothing in the Meetings article suggesting that licences impact the limits.  The only mention of licensing is the need to hold a certain license to organise meetings, webinars or live events.  For customer clarity this exception should be called out.

**Comment by get-itips** [comment]
> I disagree.  The limits and specifications article is only linked at the bottom as further reading.  There is nothing in the Meetings article suggesting that licences impact the limits.  The only mention of licensing is the need to hold a certain license to organise meetings, webinars or live events.  For customer clarity this exception should be called out.

In the past experience, when proposed to repeat information in more than one article, it was rejected as it is preferred to keep one source of information, I'll check with article author on the decision on this case and get back

**Comment by MicrosoftHeidi** [comment]
I'm the owner of this article, and we should leave things as-is. When we have dilution of information, it results in things getting out of sync. There is a link to further reading and that's sufficient. We will not change this article.
